SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1694,11 +1701,20 @@ void CNEMOEulerSolver::SetNondimensionalization(CConfig *config, unsigned short
 
     ModelTable << config->GetGasModel();
 
+    switch(config->GetKind_FluidModel()){
```

Comment:
When running, do the headers match up for these?  I know I had some issues in the past

Reply:
yup, there was a mismatch before but I fixed it in one of the commits